ddpui/services/chart_service.py

```
@@ -0,0 +1,354 @@
+
+        Returns:
+            Updated Chart instance
+
+        Raises:
+            ChartNotFoundError: If chart doesn't exist
+            ChartValidationError: If updated configuration is invalid
+        """
+        chart = ChartService.get_chart(chart_id, org)
+
+        # Prepare updated values for validation
+        updated_chart_type = chart_type if chart_type is not None else chart.chart_type
```

Ruff (0.14.8)

Local variable `updated_chart_type` is assigned to but never used

Remove assignment to unused variable `updated_chart_type`

(F841)